src/main/kotlin/eu/europa/ec/eudi/openid4vci/internal/http/AuthorizationEndpointClient.kt

```diff
@@ -196,7 +199,14 @@ internal class AuthorizationEndpointClient(
                 }
             }
             if (credentialsAuthorizationDetails.isNotEmpty()) {
-                authorizationDetails(credentialsAuthorizationDetails.map(::toNimbus))
```

Comment:
Why method was removed?

Reply:
Is moved to AuthorizationDetailsOps.kt along with other utility methods that have to do with how authorization_details are embeded as form parameter